IntegrationTestSuite/WebservicesIntegrationTests/Tests/EngineeringModel/EngineeringModel/EngineeringModelTestFixture.cs

```diff
@@ -0,0 +1,106 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="EngineeringModelTestFixture.cs" company="RHEA System S.A.">
+//
+//   Copyright 2016-2021 RHEA System S.A.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace WebservicesIntegrationTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Linq;
+
+    using Newtonsoft.Json.Linq;
+
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class EngineeringModelTestFixture : WebClientTestFixtureBase
+    {
```

Comment:
please also create a new engineeringmodel and verify you get more than 1 back...